pkg/network/node/node.go

```diff
@@ -146,10 +144,6 @@ func New(c *OsdnNodeConfig) (*OsdnNode, error) {
 		return nil, fmt.Errorf("Unknown plugin name %q", networkInfo.PluginName)
 	}
 
-	if useConnTrack && c.ProxyMode == kubeproxyconfig.ProxyModeUserspace {
-		return nil, fmt.Errorf("%q plugin is not compatible with proxy-mode %q", networkInfo.PluginName, c.ProxyMode)
-	}
```

Comment:
You should remove `ProxyMode` from `OsdnNodeConfig`, since it no longer varies in any way that matters to the sdn node controller.